WcaOnRails/app/models/competition.rb

```diff
@@ -824,6 +824,25 @@ def to_wcif
     }
   end
 
+  def set_wcif_events!(wcif_events)
+    # Remove extra events.
```

Comment:
Unfortunately, we cannot nuke and recreate all the `competition_events` for this competition, because it would break foreign keys. For instance, the [`registration_competition_events` join table](https://github.com/jfly/worldcubeassociation.org/blob/c470fe4ce7ed79f0ba343888fb443233248bad97/WcaOnRails/db/structure.sql#L793) has a `competition_event_id` foreign key in it.

We currently do not have anything that refers to rounds by foreign key, but I cannot easily anticipate if we'll have such a thing in the future. I'm worrying about this because you convinced me to change the round updating to do a "nuke and replace", but that would break round foreign keys if we ever have any in the future. What do you think is best to do here?